src/common.js

```diff
@@ -109,7 +109,8 @@ export function each(value, cb) {
 }
 
 export function isEnumerable(base, prop) {
-    return Object.getOwnPropertyDescriptor(base, prop).enumerable
+    const descriptor = Object.getOwnPropertyDescriptor(base, prop)
+    return descriptor && descriptor.enumerable
```

Comment:
Was this causing a crash? I don't think we ever pass a property that doesn't exist. Or is this just for posterity?

Reply:
The descriptor is going to be undefined for Maps here https://github.com/immerjs/immer/blob/v3.1/src/immer.js#L288